assistants/prospector-assistant/assistant/chat.py

```diff
@@ -151,24 +140,16 @@ async def on_chat_message_created(
 
     # update the participant status to indicate the assistant is thinking
     async with send_error_message_on_exception(context), context.set_status("thinking..."):
-        config = await assistant_config.get(context.assistant)
-
-        metadata: dict[str, Any] = {"debug": {"content_safety": event.data.get(content_safety.metadata_key, {})}}
-
         #
         # NOTE: we're experimenting with agents, if they are enabled, use them to respond to the conversation
         #
+        config = await assistant_config.get(context.assistant)
+        metadata: dict[str, Any] = {"debug": {"content_safety": event.data.get(content_safety.metadata_key, {})}}
 
-        # if config.agents_config.document_agent.enabled:  # To do... tie into config.
-        global is_doc_agent_running
-        if is_doc_agent_running:
-            is_doc_agent_running = await document_agent_respond_to_conversation(config, context, message, metadata)
-            return
-
-        await form_fill_execute(context, message)
-
-        # # Prospector assistant response
-        # await respond_to_conversation(context, config, message, metadata)
+        if config.guided_workflow == "Form Completion":
```

Comment:
I'd invert this since form completion is our desired default.  Maybe consider using a match w/ the form completion as the default.

Reply:
ohh i like the match idea